specification/security/create_cross_cluster_api_key/CreateCrossClusterApiKeyRequest.ts

```diff
@@ -0,0 +1,73 @@
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Access } from '@security/_types/Access'
+import { RequestBase } from '@_types/Base'
+import { Metadata, Name } from '@_types/common'
+import { Duration } from '@_types/Time'
+
+/**
+ * Create a cross-cluster API key.
+ *
+ * Create an API key of the `cross_cluster` type for the API key based remote cluster access.
+ * A `cross_cluster` API key cannot be used to authenticate through the REST interface.
+ *
+ * IMPORTANT: To authenticate this request you must use a credential that is not an API key. Even if you use an API key that has the required privilege, the API returns an error.
+ *
+ * Cross-cluster API keys are created by the Elasticsearch API key service, which is automatically enabled.
+ *
+ * NOTE: Unlike REST API keys, a cross-cluster API key does not capture permissions of the authenticated user. The API key’s effective permission is exactly as specified with the `access` property.
+ *
+ * A successful request returns a JSON structure that contains the API key, its unique ID, and its name. If applicable, it also returns expiration information for the API key in milliseconds.
+ *
+ * By default, API keys never expire. You can specify expiration information when you create the API keys.
+ *
+ * Cross-cluster API keys can only be updated with the update cross-cluster API key API.
+ * Attempting to update them with the update REST API key API or the bulk update REST API keys API will result in an error.
+ * @rest_spec_name security.create_cross_cluster_api_key
+ * @availability stack stability=stable
+ * @cluster_privileges manage_security
+ * @ext_doc_id remote-clusters-api-key
+ */
+export interface Request extends RequestBase {
+  body: {
+    /**
+     * The access to be granted to this API key.
+     * The access is composed of permissions for cross-cluster search and cross-cluster replication.
+     * At least one of them must be specified.
+     *
+     * NOTE: No explicit privileges should be specified for either search or replication access.
+     * The creation process automatically converts the access specification to a role descriptor which has relevant privileges assigned accordingly.
+     */
+    access: Access
+    /**
+     * Expiration time for the API key.
+     * By default, API keys never expire.
+     */
+    expiration?: Duration
+    /**
+     * Arbitrary metadata that you want to associate with the API key.
+     * It supports nested data structure.
+     * Within the metadata object, keys beginning with `_` are reserved for system usage.
+     */
+    metadata?: Metadata
+    /** Specifies the name for this API key. */
+    name?: Name
```

Comment:
`name` is required

Reply:
Thanks, fixed in d5d06c140a94be606f486d570f1579d734add9eb